src/core/namespace.c

```diff
@@ -99,6 +99,7 @@ static const MountEntry protect_kernel_tunables_table[] = {
         { "/sys/kernel/debug",   READONLY,     true  },
         { "/sys/kernel/tracing", READONLY,     true  },
         { "/sys/fs/cgroup",      READWRITE,    false }, /* READONLY is set by ProtectControlGroups= option */
+        { "/sys/fs/selinux",     READWRITE,    true  },
```

Comment:
Oh, this affects all services, so it doesn't matter whether `HAVE_SELINUX` is defined or not. Sorry for the noise.